src/imitation/algorithms/dagger.py

```diff
@@ -310,7 +314,10 @@ def _try_load_demos(self):
             logging.info(
                 f"Loaded {sum(num_demos)} new demos from {len(num_demos)} rounds"
             )
-            self.bc_trainer.set_expert_dataset(transitions)
+            dataloader = th_data.DataLoader(
+                transitions, self.batch_size, drop_last=True
+            )
```

Comment:
We could make this more customizable beyond `batch_size` but this seems like a reasonable default to start with. (Equivalent to behavior before this commit, except without shuffling.)

- [ ] Add `shuffle=True`.